capa/ida/plugin/form.py

```diff
@@ -251,8 +283,12 @@ def load_interface(self):
         self.load_view_tree_tab()
         self.load_view_rulegen_tab()
         self.load_view_status_label()
+        self.load_result_cache_label()
         self.load_view_buttons()
 
+        # reset on tab change program analysis/rule generator
+        self.view_tabs.currentChanged.connect(lambda s: self.set_view_result_cache_label(""))
+
```

Comment:
What about when user navigates from program analysis to rule generator and back?

Reply:
moving this to #1305, I think it's fine as is, but can be improved